analytics/components/Plausible.tsx

```diff
@@ -0,0 +1,104 @@
+
+const plausibleScript = exclusionScript;
+
+const sendEvent = (
+  _action: string,
+  _type: string,
+  event: { name?: string; params?: Record<string, string> },
+) => {
+  const origEvent = event?.name;
+  const ecommerce = event?.params;
+
+  if (origEvent && ecommerce) {
+    const flagsObject = {} as Record<string, boolean>;
+    if (window.LIVE.flags) {
+      for (const flag of window.LIVE.flags) {
+        if (flag.name && flag.value) {
+          flagsObject[flag.name] = flag.value;
+        }
+      }
+    }
+    const values = {} as Record<string, string>;
+    for (const key in ecommerce) {
+      if (ecommerce[key] !== null && ecommerce[key] !== undefined) {
+        values[key] = (typeof ecommerce[key] !== "object")
+          ? ecommerce[key]
+          : JSON.stringify(ecommerce[key]);
+      }
+    }
+    window.plausible(origEvent, { props: { ...flagsObject, ...values } });
+  }
+};
+
+function Component({
+  async,
+}: Props) {
+  if (!context.isDeploy) {
+    return <></>;
+  }
+  return (
+    <>
+      <Head>
+        <link rel="dns-prefetch" href="https://plausible.io/api/event" />
+        <link
+          rel="preconnect"
+          href="https://plausible.io/api/event"
+          crossOrigin="anonymous"
+        />
+        {async &&
+          (
+            <Script
```

Comment:
This Script that render Head, inside another Head, wasn't working, have you tested?

Reply:
I will remove async usage (not really used today)

Reply:
Removed from plausible.